src/client/app/strings/german.ts

```diff
@@ -114,8 +116,8 @@ export function getGermanStrings(): ClientStrings {
         shareMenuPublic: 'Neues öffentliches GitHub-Gist',
         shareMenuPrivate: 'Neues geheimes GitHub-Gist',
         updateMenu: 'Bestehendes GitHub-Gist aktualisieren',
-        sharePublicSnippetConfirm: 'Sind Sie sicher, dass dieses Schnipsel in einem neuen öffentlichen GitHub-Gist geteilt werden soll?',
-        sharePrivateSnippetConfirm: 'Sind Sie sicher, dass dieses Schnipsel in einem neuen geheimen GitHub-Gist geteilt werden soll?',
+        sharePublicSnippetConfirm: englishSubstitutesForNotYetTranslated.sharePublicSnippetConfirm,
```

Comment:
Instead of deleting the old, can you comment them out, and add a comment for Mourad to know why he needs to update them (and also what the old text was, and what changed)?